_rules/video-only-description-track-ac7dc6.md

```diff
@@ -24,7 +24,7 @@ acknowledgments:
 
 ## Applicability
 
-The rule applies to any [non-streaming](#non-streaming-media-element) `video` element [visible][] where the video doesn't contain audio and does contain a `track` element with a `kind="descriptions"` attribute.
+The rule applies to every [non-streaming](#non-streaming-media-element) `video` element that is [visible][] where the video contains audio and a `track` element with a `kind` [attribute value][] of `descriptions`.
```

Comment:
You have changed the meaning of the applicability from videos with no audio to videos with audio. I guess that was a mistake (and certainly out of the scope of this PR)
```suggestion
The rule applies to every [non-streaming](#non-streaming-media-element) `video` element that is [visible][] where the video does not contain audio and contains a `track` element with a `kind` [attribute value][] of `descriptions`.
```